modules/ossm-mixer-policy.adoc

```diff
@@ -10,6 +10,7 @@ In previous versions of {ProductName}, Mixer’s policy enforcement was enabled
 .Prerequisites
 * Access to the {product-title} Command-line Interface (CLI) also known as `oc`.
 
+NOTE: `istio-system` is the default namespace the Service Mesh Control Plane (SMCP) installs to. You can change `istio-system` to any other name during the installation.
```

Comment:
`istio-system` is not exactly the default. In the docs, we give istio system as the exampel as the control plane namespace. You could choose anything. But I'm not sure that you can change the name of it to anything else during the installation.

I think it would be more consistent with some changes I've recently made to say in step 2: 

>In this example `istio-system is the name of the `ServiceMeshControlPlane` namespace.

Reply:
See my comment above about perhaps using angle brackets,   \<istio-system>  in the commands.